packages/bounds/src/server.ts

```diff
@@ -0,0 +1,13 @@
+import type * as API from "./index";
```

Comment:
Since you cannot possibly get an element, wouldn't it be better to use nulledBounds from before for the server? Or have an option for that?